.github/workflows/update-israeli-bank-scrapers.yml

```diff
@@ -0,0 +1,79 @@
+name: Update israeli-bank-scrapers
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  # Manual trigger
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: false
+        default: 'information'
+        type: choice
+        options:
+          - information
+          - debug
+          - warning
+          - critical
+      tags:
+        description: 'Purpose of Run This Workflow?'
+        required: false
+        type: string
```

Comment:
I can't see where you're using these inputs

Reply:
this is some inputs when you are running it manually here:
![image](https://github.com/brafdlog/caspion/assets/52237617/172139b7-4a30-4219-9a86-ca5a1ee75d49)
This can be removed depends on your opinion

Reply:
I know where we set them, but they are not used later in the workflow

Reply:
I have removed it